Makefile.dep

```diff
@@ -78,12 +78,16 @@ ifneq (,$(filter ng_ipv6_default,$(USEMODULE)))
   USEMODULE += ng_ipv6
   USEMODULE += ng_icmpv6
   USEMODULE += ng_ndp
+  USEMODULE += ng_ndp_internal
+  USEMODULE += ng_ndp_node
 endif
 
 ifneq (,$(filter ng_ipv6_router_default,$(USEMODULE)))
   USEMODULE += ng_ipv6_router
   USEMODULE += ng_icmpv6
   USEMODULE += ng_ndp
+  USEMODULE += ng_ndp_internal
+  USEMODULE += ng_ndp_node
```

Comment:
No, that would pull in `ng_ndp_host`. Host and router are distinct classes in vanilla IPv6.